.github/workflows/pr-checker.yaml

```diff
@@ -0,0 +1,37 @@
+name: pr-checker
+on: 
+  pull_request:
+env:
+  GO_VERSION: "1.16"
+jobs:
+  build-test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+        matrix:
+            os: [ubuntu-20.04]
+    steps:
+      - name: Setup Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ env.GO_VERSION }}
+      - name: Checkout Code
+        uses: actions/checkout@v2
+        with:
+          submodules: true
+      - name: Install Dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install --yes build-essential pkgconf libelf-dev llvm-12 clang-12
+          for tool in "clang" "llc" "llvm-strip"
+          do
+            sudo rm -f /usr/bin/$tool
+            sudo ln -s /usr/bin/$tool-12 /usr/bin/$tool
+          done
+          sudo curl -L -o /usr/bin/opa https://github.com/open-policy-agent/opa/releases/download/v0.35.0/opa_linux_amd64 && sudo chmod 755 /usr/bin/opa
+      - name: Build
+        run: make all
+      - name: Test tracee-ebpf
+        run: make -C tracee-ebpf/ test
```

Comment:
We need root permissions here:

https://github.com/aquasecurity/tracee/issues/1225

Reply:
I see now ... Fundamentally it's no longer a unit test. It's kind of an integration tests that runs along with other unit tests.

Reply:
No longer relevant in scope of this PR. Anyway we have #1228 to follow up